### ISSUE TITLE
add support for datetimer picker

### ISSUE THREAD

**Issue by Gillardo** [angular-ui/bootstrap#3379]
I love the datepicker and timepicker in bootstrap-ui but in some projects you need to pick both and need to use one input.  I have created a wrapper for datepicker and timepicker called datetime-picker which is based on the code from datepicker-popup.

It would be nice to have this included as standard as sure i aint the only one that requires this functionality.

Here is my github project

https://github.com/Gillardo/bootstrap-ui-datetime-picker

**Comment by rvanbaalen** [comment]
Thanks @Gillardo for suggesting this enhancement. We will consider implementing similar functionality. 

The datepicker itself first needs some extra love and attention (bug fixes) before we start implementing new features though.

Closing for now. But we will find it when the time comes (it has the correct labels).
